src/cephadm/cephadmlib/daemons/smb.py

```diff
@@ -200,6 +203,14 @@ def args(self) -> List[str]:
         return super().args() + ['update-config', '--watch']
 
 
+class SmbmetricsContainer(SambaContainerCommon):
+    def name(self) -> str:
+        return 'smbmetrics'
+
+    def args(self) -> List[str]:
+        return []
```

Comment:
This looks weird and I had to look at the base class to see that there was a legit reason for it. I think it might be better to split the base class into something like `ContainerCommon` and then have `SambaContainerCommon` inherit from that, then this new class would inherit directly from `ContainerCommon` since it doesn't use images from samba-container project.

Also naming nitpick: 'SmbmetricsContainer' -> 'SMBMetricsContainer'.

Reply:
Fair enough -- will fix.